src/vs/platform/window/common/window.ts

```diff
@@ -177,7 +178,7 @@ export function useWindowControlsOverlay(configurationService: IConfigurationSer
 		return configuredUseWindowControlsOverlay;
 	}
 
-	return false; // disable by default
+	return productService.quality === 'insider' || productService.quality === 'exploration';
```

Comment:
do we need this check? the default value in the setting should be enough no?

Reply:
To reduce bloat I have re-enabled it on all builds.
On OSS, the WCO will render on the wrong side until VS Code moves onto Electron 22, but that shouldn't be too big of a problem.